lib.js

```diff
@@ -1,32 +1,32 @@
-const {
```

Comment:
Why do you prefer the explicit require per function instead of block require?

Reply:
It was loading 12M of size if we load from utilities, but when i was loading from each one separate, each one was of size around 10-20k only.